ivy/functional/frontends/tensorflow/tensor.py

```diff
@@ -228,6 +228,70 @@ def __iter__(self):
             yield self[i]
 
 
+class TensorShape:
+    # TODO: there are still some methods that may need implementing
+
+    def __init__(self, dims):
+        self._dims = tuple(dims)
+
+    def __repr__(self):
+        return str(self._dims)
```

Comment:
Is this really the original implementation? I think we should be printing something like this 
![image](https://github.com/unifyai/ivy/assets/111577222/57e05854-11cb-4af5-a19d-391c64313b41)

Reply:
Yeah you're right, just looked at the source code. For some reason when I ran it in Python locally it returned the shape in the tuple format which is why I did it like this, not sure what was causing that